pairing/bn256/suite_test.go

```diff
@@ -406,43 +406,3 @@ func TestNegPairAll(t *testing.T) {
 	require.True(t, pair2.Equal(pair3))
 	require.True(t, pair3.Equal(pair4))
 }
-
-func BenchmarkBn256(b *testing.B) {
```

Comment:
Why deleting it?

Reply:
Somehow it got deleted in the merge, not sure why